### PR TITLE
Setup project file

### PR DESCRIPTION
**Pull request by noelfoka**
In this brancg, I did:

- [ ] Added HTML.
- [ ] Added CSS.
- [ ] Edit readme.
- [ ] Added linters.

### CODE REVIEW

**Review by frank1738 — COMMENTED** [review]
Hi @noelfoka , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [x] You have submitted a project with no readme file. Kindly use [this](https://github.com/microverseinc/readme-template) readme file and customize it to contain the project information

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by shella12 — COMMENTED** [review]
Hi @noelfoka , 

#  Status: Invalidated :red_circle: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have a professional README. As mentioned by the previous reviewer please use this [template ](https://github.com/microverseinc/readme-template)link and use the [README template](https://github.com/microverseinc/readme-template) for your README. Also don't forget to customise it by adding details about your project.


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @noelfoka 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is failing some linter checks. please follow this [guide](https://github.com/microverseinc/linters-config/tree/master/html-css) to set it up.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+#nav nav {
+  width: 45px;
+  height: 30px;
+  display: flex;
+  align-items: center;
+}
+
+section {
+  background-image:
```
Comment (topeogunleye) [inline style.css:34]: - [ ] [OPTIONAL] It's possible to download and use the background image at once. see a demo below.

https://www.loom.com/share/371616c731d14be1abfdda6805264807

File: style.css [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+  font-weight: bold;
+  font-size: 40px;
+  color: #172b4d;
+  line-height: 52px;
+  text-align: start;
+  margin: 0 24px;
+}
+
+section p {
+  font-family: Poppins, sans-serif;
```
Comment (topeogunleye) [inline style.css:76]: - [ ] [OPTIONAL] Poppins is the only font family used in this template, so it's best to declare it in your body selector to keep your css dry, as it'll be inherited from the body selctor to the other selectors.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+    url("../images/Rectangle 4.png"),
+    url("../images/Ellipse\ 3.png"),
+    url("../images/Ellipse\ 8.png"),
+    url("../images/Ellipse\ 5.png"),
+    url("../images/Rectangle\ 7.png"),
+    url("../images/Ellipse 12.png"),
+    url("../images/Ellipse 4.png"),
+    url("../images/Rectangle\ 5.png"),
+    url("../images/Ellipse\ 13.png"),
+    url("../images/Ellipse\ 15.png");
```
Comment (topeogunleye) [inline style.css:45]: - [ ] [OPTIONAL] It's possible to download and use the background image at once. see a demo below.

https://www.loom.com/share/371616c731d14be1abfdda6805264807

File: style.css [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+  font-family: Poppins, sans-serif;
+  font-weight: bold;
+  font-size: 40px;
+  color: #172b4d;
+  line-height: 52px;
+  text-align: start;
+  margin: 0 24px;
+}
+
+section p {
```
Comment (topeogunleye) [inline style.css:75]: - [ ] [OPTIONAL] Poppins is the only font family used in this template, so it's best to declare it in your body selector to keep your css dry, as it'll be inherited from the body selctor to the other selectors.

**Review by topeogunleye — APPROVED** [review]
### **STATUS: APPROVED  ✔️** 
------
Hi @noelfoka,  

Your project is complete! Great job for nailing it right on the first attempt 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟

wiserhood-cheers

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good description :heavy_check_mark:

## [Optional] suggestions:
 * [ ]  Please check my inline comments.

 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear
Please, remember to tag @topeogunleye in your question so I can receive the notification

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 3 above)
Comment (noelfoka) [reply style.css:45]: hank you very much

Comment (topeogunleye) [reply style.css:45]: > hank you very much

@noelfoka, you're welcome.

**Pull request merged**